samples/kubernetes/end2end/demo-domains/domainBuilder/scripts/simple-topology.yaml

```diff
@@ -0,0 +1,89 @@
+domainInfo:
+    AdminUserName: '@@PROP:ADMIN_USER@@'
+    AdminPassword: '@@PROP:ADMIN_PWD@@'
+    ServerStartMode: '@@PROP:SERVER_START_MODE@@'
+topology:
+    Name: '@@PROP:DOMAIN_NAME@@'
+    AdminServerName: '@@PROP:ADMIN_NAME@@'
+    ProductionModeEnabled: '@@PROP:PRODUCTION_MODE_ENABLED@@'
+    Cluster:
+        '@@PROP:CLUSTER_NAME@@':
+            ClientCertProxyEnabled: true
+            DynamicServers:
+                ServerTemplate: template1
+                CalculatedListenPorts: false
+                ServerNamePrefix: '@@PROP:MANAGED_SERVER_NAME_BASE@@'
+                DynamicClusterSize: '@@PROP:CONFIGURED_MANAGED_SERVER_COUNT@@'
+                MaxDynamicClusterSize: '@@PROP:CONFIGURED_MANAGED_SERVER_COUNT@@'
+    Server:
+        '@@PROP:ADMIN_NAME@@':
+            ListenPort: '@@PROP:ADMIN_PORT@@'
+            NetworkAccessPoint:
+                T3Channel:
+                    ListenAddress: None
+                    ListenPort: '@@PROP:T3_CHANNEL_PORT@@'
+                    PublicAddress: '@@PROP:T3_PUBLIC_ADDRESS@@'
+                    PublicPort: '@@PROP:T3_CHANNEL_PORT@@'
+    ServerTemplate:
+        template1:
+            Cluster: '@@PROP:CLUSTER_NAME@@'
+            ListenPort: '@@PROP:MANAGED_SERVER_PORT@@'
+resources:
+    JDBCSystemResource:
+        Generic1:
+            Target: '@@PROP:CLUSTER_NAME@@'
+            JdbcResource:
+                JDBCDataSourceParams:
+                    JNDIName: dc1
+                    GlobalTransactionsProtocol: OnePhaseCommit
+                JDBCDriverParams:
+                    DriverName: org.gjt.mm.mysql.Driver 
+                    URL: 'jdbc:mysql://mysql.default.svc.cluster.local:3306/@@PROP:DOMAIN_NAME@@'
+                    PasswordEncrypted: welcome1
```

Comment:
We should not hard-code usernames & passwords into WDT templates.  Minimally, they belong in a WDT prop.  Ideally, they should be specified via a config-override...